scripts/onboarding/aks/onboarding-msi-terraform/main.tf

```diff
@@ -0,0 +1,75 @@
+resource "azurerm_resource_group" "rg" {
+  name     = var.aks_resource_group_name
+  location = var.resource_group_location
+}
```

Comment:
why cant we merge with and without syslog to single terraform  templates?

Reply:
in that case the resource will in this format: 
resource "example_resource" "resource_with_syslog" {
  count = var.syslog ? 1 : 0
  name = "Resource with Syslog"
}
resource "example_resource" "resource_without_syslog" {
  count = var.syslog ? 0 : 1
  name = "Resource without Syslog"
}